src/httpclient.cc

```diff
@@ -111,30 +124,50 @@ bool HttpClient::authenticate(const AuthConfig& conf) {
   headers = curl_slist_append(headers, "Content-Type: application/json");
   headers = curl_slist_append(headers, "charsets: utf-8");
   curl_easy_setopt(curl, CURLOPT_HTTPHEADER, headers);
-
+  authenticated = true;
   return true;
 }
 
-Json::Value HttpClient::get(const std::string& url) {
+std::string HttpClient::get(const std::string& url) {
   curl_easy_setopt(curl, CURLOPT_URL, url.c_str());
   return perform(curl);
 }
 
+Json::Value HttpClient::getJson(const std::string& url) {
+  Json::Value json;
+  Json::Reader reader;
+
+  std::string response = get(url);
+  reader.parse(response, json);
+  return json;
+}
+
 Json::Value HttpClient::post(const std::string& url, const Json::Value& data) {
-  return post(url, Json::FastWriter().write(data));
+  Json::Value json;
+  Json::Reader reader;
+
+  std::string response = post(url, Json::FastWriter().write(data));
+  reader.parse(response, json);
+  return json;
+}
+
+void HttpClient::setCerts(const std::string& ca, const std::string& cert) {
+  curl_easy_setopt(curl, CURLOPT_SSL_VERIFYPEER, false);
```

Comment:
Is this needed?  Disabling CURLOPT_SSL_VERIFYPEER will allow man-in-the-middle attacks against the SSL connection